.github/dependabot.yaml

```diff
@@ -4,3 +4,7 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Considering we have a daily cron for gomod, should we run daily for github-actions too?

Reply:
Sounds good to me! Updated in 2761ecdee07cd8bc7ce816b5902d150938140906